### DISCUSSION TITLE
Links between Tickets and Discussions are not shown (in the opposite direction)

### DISCUSSION THREAD

**Discussion by RubenKelevra** [community/community#3211] [Discussions]
When I link a ticket inside another ticket, I'll get a cross-link - (aka pingback) that this happened in the other ticket.

It would be super useful if this could also happen between discussions and tickets. When I mention a discussion in a ticket, I like see a small hint that this happened in the discussion as well as the other way around.

**Comment by lesteve** [comment]
I agree that automated back-links (not sure what the exact technical term is) is useful to find related issues and get a better sense of the general context. I feel like not being able to find related Discussions is currently missing in Github Discussions.

I seem to remember at one point I found a Discussion that was saying that an alternative design was under consideration for Github Discussions (maybe a related issues tab in the right hand-side panel) but I can not find it anymore ... I do agree that sometimes automated back-links can be a bit noisy or hard to find if they are spread over a long issue.

Just to try to be as clear as possible here is an example of what I mean by automated back-links:
- https://github.com/scikit-learn/scikit-learn/issues/13621#issuecomment-483093642 in issue `#13621` links to issue `#9413`
![image](https://user-images.githubusercontent.com/1680079/107190714-6ead1500-69eb-11eb-9f7b-7d5d4b57fbd9.png)
- in issue `#9413` an automated link is generated
![image](https://user-images.githubusercontent.com/1680079/107190544-3574a500-69eb-11eb-9608-d0f0b5da0b1d.png)

**Comment by corneliusroemer** [comment]
Back-links are the major thing I miss from discussions when compared to issues. It really makes discussions feel more isolated and less connected.

**Comment by marcindulak** [comment]
I believe this was already requested in https://github.com/orgs/community/discussions/2971

**Reply by michael-hawker** [reply]
2971 has more discussion and upvotes/thumbs, so it's weird to see it closed over this one, generally you want to keep the oldest issue open for tracking. This one should be closed as the duplicate.

**Comment by irh** [comment]
#2971 was closed in favour of this discussion, so I'll say again here that automatic links between Discussions and Issues/PRs would be very helpful, it still feels strange to have discussions disconnected from the rest of the system.

**Comment by oriolpiera** [comment]
Hi folks,

A new one in this party.

I already open a discussion (because not related discussion appeared) and inmediatly closed it as seen both previous discussions and the weird thing to close the one-month oldest one.

https://github.com/orgs/community/discussions/125524

I take a seat.

**Comment by agneszitte** [comment]
**Big +1** as the others, I saw that https://github.com/orgs/community/discussions/2971 is closed (which had a larger number of votes and was mentioned multiple times). 

It is very useful to have links between Issues and Pull Requests, but Discussions are challenging to use without proper links. 
This leads to duplicate content, questions, or comments because some people only look at issues, some at discussions, and few view everything globally on some repositories. 

Additionally, considering that you currently can't add discussions to the project board, it undermines the real purpose of having discussions in terms of better project management (related issue: https://github.com/orgs/community/discussions/11223).

In the end, you can see that it doesn't even facilitate discussions within the GitHub community itself when similar topics are mentioned in different discussions/issues on https://github.com/orgs/community/discussions.
